docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: p0stgr3s_pass
+    ports: 
+      - '5432:5432'
```

Comment:
Is it necessary to expose this port? The Calendso container will already find the database because of Docker's internal routing.

Reply:
it's a best practice to document them, for what it's worth. It also helps if you want to use an external postgresql client.

Reply:
I'd recommend leaving it there but commented out as it could be a security issue for anyone who adds the compose file to their server and exposes their db port by accident. Docker exposed ports often bypass firewalls too!